TpmTestingPkg/TpmReplayPei/Tool/TpmReplay.py

```diff
@@ -579,6 +592,76 @@ def _add_digest(pcr: int, alg: int, digest: bytes, prehashed: bool = False) -> N
 
     return replay_event_log
 
+def _decode_efi_signature_data_x509(data):
+    cert = x509.load_der_x509_certificate(data)
+    cert_info = {
+        str(cert.subject): {
+            "fingerprint": cert.fingerprint(hashes.SHA1()).hex(":"),
+            "issuer": str(cert.issuer),
+            "not_valid_after_utc": str(cert.not_valid_after_utc),
+        }
+    }
+
+    return cert_info
+
+def _decode_efi_signature_data_sha256(signature) -> dict:

```

Comment:
Please use `Dict` for the type hint with the dictionary types indicated (e.g. `Dict[str, str]` used elsewhere in the file).